docs/Clearwater_Architecture.md

```diff
@@ -35,7 +35,7 @@ Dime nodes run Clearwater's Homestead and Ralf components.
 
 #### Homestead (HSS Cache)
 
-Homestead provides a web services interface to Sprout for retrieving authentication credentials and user profile information.  It can either master the data (in which case it exposes a web services provisioning interface) or can pull the data from an IMS compliant HSS over the Cx interface.  The Homestead nodes themselves are stateless - the mastered / cached subscriber data is all stored on Vellum (Cassandra for the mastered data, and Astaire/Memcached for the cached data).
+Homestead provides a web services interface to Sprout for retrieving authentication credentials and user profile information.  It can either master the data (in which case it exposes a web services provisioning interface) or can pull the data from an IMS compliant HSS over the Cx interface.  The Homestead nodes themselves are stateless - the mastered / cached subscriber data is all stored on Vellum (Cassandra for the mastered data, and Astaire/Rogers/Memcached for the cached data).
```

Comment:
I know this is an existing issue, but it feels like this section is referring to Homestead as a node, not a service.

Can we replace:

> It can either master the data (in which case it exposes a web services provisioning interface)

with something like:

> It can either master the data (with a web services provisioning interface provided by Homestead-Prov)